cset-workflow/meta/diagnostics/rose-meta.conf

```diff
@@ -51,25 +51,35 @@ type=python_boolean
 compulsory=true
 trigger=template variables=LATITUDE_POINT: True;
         template variables=LONGITUDE_POINT: True;
+        template variables=LATLON_IN_TYPE: True;
         template variables=SINGLE_POINT_METHOD: True;
 sort-key=0surface5
 
 [template variables=LATITUDE_POINT]
 ns=Diagnostics/Quicklook
-description=Latitude of selected point in the same coordinate system as the data.
+description=Latitude of selected point.
 help=The latitude must exist within the domain. Value should be a float: for example, -1.5.
 type=real
 compulsory=true
 sort-key=0surface6
 
 [template variables=LONGITUDE_POINT]
 ns=Diagnostics/Quicklook
-description=Longitude of selected point in the same coordinate system as the data.
+description=Longitude of selected point.
 help=The longitude must exist within the domain. Value should be a float: for example, 0.8.
 type=real
 compulsory=true
 sort-key=0surface6
 
+[template variables=LATLON_IN_TYPE]
+ns=Diagnostics/Quicklook
+description=Method used to map model data onto selected gridpoints.
+help=This switch indicates whether the selected latitude longitude coordinates are on the real world grid or
+    the rotated grid specified by the input data.
+values="realworld", "rotated"
+compulsory=true
+sort-key=0surface6
```

Comment:
In what cases would we not want to use real world coordinates? I'm wondering if we need this to be configurable, or if we should just use them all the time.